src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -1,20 +1,60 @@
-import React, { useState } from 'react';
+import React, { useState, useContext, useMemo } from 'react';
 import {
   Button, Stepper, FullscreenModal, Container,
 } from '@edx/paragon';
+import algoliasearch from 'algoliasearch/lite';
+import { Configure, InstantSearch } from 'react-instantsearch-dom';
+import { getConfig } from '@edx/frontend-platform/config';
+import { SearchContext } from '@edx/frontend-enterprise-catalog-search';
+import FacetListRefinement from '@edx/frontend-enterprise-catalog-search/FacetListRefinement';
```

Comment:
FacetListRefinement and CurrentRefinements are not currently exporting from `edx/frontend-enterprise-catalog-search` that's why they are imported this way. I have created a PR to make them available as a simple import.

Reply:
Just to link to that PR: https://github.com/edx/frontend-enterprise/pull/138

Also, note that publishing in the `frontend-enterprise` monorepo is not quite fully automated. There is still an issue where Lerna attempts to publish over an existing NPM version, causing the publish to fail. When this happens, you'll need to follow the steps outlined in this how-to doc: https://github.com/edx/frontend-enterprise/blob/master/docs/how_tos/resolving_publishing_issues_with_lerna.rst

Definitely let me know if you have any questions on the publishing here 😄

Reply:
Thank you so much for sharing this.

Reply:
I'm merging this PR and will change this import once this [PR](https://github.com/edx/frontend-enterprise/pull/139) is merged.